src/settings/profile.hpp

```diff
@@ -101,7 +101,41 @@ class Profile
 
         if (game == btu::Game::FNV)
```

Comment:
Maybe this should be moved to a separate function

Reply:
Agreed, I just didn’t figure out a place where it would make the most amount of sense.